src/app/lib/utilities/parseAssetData/index.js

```diff
@@ -1,12 +1,26 @@
 import path from 'ramda/src/path';
+import pathOr from 'ramda/src/pathOr';
 
 const getISOStringDate = date => new Date(date).toISOString();
 
 export const getArticleId = path(['metadata', 'id']);
 
 export const getHeadline = path(['promo', 'headlines', 'seoHeadline']);
 
-export const getSummary = path(['promo', 'summary']);
+export const getSummary = pathOr('', [
```

Comment:
Allowing `getSummary` to return an empty string as opposed to `undefined` could lead to bugs, because an empty string is also a legitimate value for the summary and it's good to be able to differentiate between the two.

Can I suggest we follow the example of the other functions and use `ramda/src/path`, which will return `undefined` and then when we call `getSummary` short-circuit with `||` to set a default value when it's not found?

Reply:
@simonsinclair Agreed with the approach to just use path here. This helps since in the Article page logic when the summary is undefined, we should use the headline instead: https://github.com/bbc/simorgh/blob/5005a6e43adc86c996cebcbd4c43d25c8d525b83/src/app/pages/ArticlePage/index.jsx#L93